src/helper/module.js

```diff
@@ -1,3 +1,4 @@
 export { CallbackType } from './CallbackType';
 export { CallbackHelper } from './CallbackHelper';
 export { RequestHelper } from './RequestHelper';
+export { timeout } from './timeout';

```

Comment:
Man muss nicht jede Kleinigkeit in ein Paket auslagern.. der Code, der zum Einbinden generiert wird ist länger, als der Code im Paket. Einfach direkt an die jeweiligen (nur zwei!) Stellen einfügen

Reply:
Da es keine direkte Funktionalität der TextControlContext klasse ist, habe ich alle Helper Funktionen in diesem Modul ausgelagert. 
Sehe es an dieser Stelle schon als richtig positioniert an, auch wenn es nur eine kleine Funktion ist.